src/assets/svg/bridge-logo-large.svg

```diff
@@ -1,25 +1,11 @@
-<path d="M118.653 5.90928C119.233 5.87859 119.808 6.01956 120.308 6.31449C120.807 6.60943 121.208 7.04521 121.461 7.56717C122.306 7.76973 123.117 7.97229 123.928 8.17549V4.62747H121.9V7.26397C121.644 6.43613 121.127 5.71296 120.427 5.20168C119.727 4.6904 118.881 4.41824 118.014 4.42556C117.294 4.42247 116.585 4.59653 115.948 4.93235C115.312 5.26818 114.768 5.75545 114.364 6.35116C115.108 6.41868 115.851 6.51997 116.595 6.655C116.869 6.39684 117.193 6.19749 117.547 6.06924C117.901 5.94099 118.278 5.88655 118.653 5.90928Z" fill="#29707A"/>
-<path d="M131.226 14.4578C129.401 14.4578 128.184 13.1741 128.015 10.8098C127.339 10.5395 126.663 10.3034 125.987 10.0667V10.1679C125.987 13.5121 127.914 15.9441 131.226 15.9441C131.918 15.982 132.61 15.8788 133.261 15.6407C133.912 15.4026 134.507 15.0346 135.011 14.5591C134.538 14.2211 134.031 13.8837 133.527 13.5456C133.234 13.8581 132.876 14.102 132.479 14.2597C132.081 14.4175 131.653 14.4851 131.226 14.4578V14.4578Z" fill="#29707A"/>
-<path d="M136.398 9.99867C136.398 6.58677 134.302 4.39087 131.227 4.39087C128.422 4.39087 126.564 6.18101 126.124 8.81621C127.799 9.36078 129.436 10.0153 131.024 10.7754H136.398C136.381 10.5167 136.381 10.2573 136.398 9.99867V9.99867ZM128.017 9.25552C128.254 7.0622 129.369 5.91136 131.193 5.91136C133.018 5.91136 134.202 7.26283 134.336 9.25552H128.017Z" fill="#29707A"/>
-<path d="M136.126 12.4653H134.064C134.623 12.7982 135.164 13.1591 135.686 13.5465C135.893 13.2136 136.042 12.848 136.126 12.4653V12.4653Z" fill="#29707A"/>
+<svg xmlns="http://www.w3.org/2000/svg" width="137" height="20" fill="none" viewBox="0 0 137 20">
+  <g class="bridge-logo-large" clip-path="url(#a)">
+    <path fill="var(--kd-color-background-logo-kyndryl-neutral-txt,#FF462D)" fill-rule="evenodd" d="M0 .1v15.71h2.03V9.83l5.24 5.98h2.5L4.36 9.63l5.178-4.88 3.842 9.84h1.96l-1.79 5.24h2.13l5.07-15.1h-2.09l-3.24 9.66-3.79-9.66H7.06L2.03 9.46V.1H0Zm31.67 15.71V8.92l-.01-.01c-.03-2.5-1.35-4.39-3.85-4.39-.77-.01-1.52.22-2.15.66-.62.45-1.09 1.08-1.33 1.81V4.73H22.3V15.8h2.03V9.25c0-2.26 1.08-3.24 2.67-3.24 1.59 0 2.64.95 2.64 3.21v6.59h2.03ZM42.38.1v7.3c-.25-.84-.77-1.57-1.47-2.09a4.07 4.07 0 0 0-2.42-.81c-2.67 0-4.7 2.36-4.7 5.74 0 3.38 2.03 5.74 4.7 5.74.83-.01 1.65-.27 2.34-.74s1.23-1.13 1.55-1.9v2.4h2.03V.1h-2.03Zm-3.25 14.43c-2.1 0-3.31-1.66-3.31-4.26 0-2.6 1.18-4.26 3.31-4.26s3.31 1.66 3.31 4.26c0 2.6-1.22 4.26-3.31 4.26Zm10.21-7.506a3.495 3.495 0 0 1 3.62-2.494v1.99h-.61c-2.03 0-3.01.64-3.01 2.63v6.65h-2.03V4.73h2.03v2.294ZM62.29 4.73h2.09l-5.07 15.1h-2.13l1.79-5.24h-1.96l-3.85-9.86h2.1l3.79 9.66 3.24-9.66ZM65.93.1h2.03v15.67h-2.03V.1Z" class="primary" clip-rule="evenodd"/>
```

Comment:
Shouldn't this token be `--kd-color-background-logo-kyndryl-txt` (red/brand color)? Same comment for small svg.

Reply:
Edit: Nevermind, I see it's black instead of red in Figma now. I think it's the fallback hex color is wrong then.

Reply:
Ux has been updated with correct token now.